### ISSUE TITLE
Plugin identified as 0.0.1

### ISSUE THREAD

**Issue by Skvare** [atlascoder/tracker_hider#4]
The newest release still identifies itself in /admin/plugins as 0.0.1 (instead of 0.1.0).

**Comment by atlascoder** [comment]
Yes, here is a mess with releases, i will fix it near days, thanks!
